src/gtc/dace/expansion.py

```diff
@@ -926,6 +926,268 @@ def add_nodes_and_edges(self):
                 tasklet.in_connectors[edge.dst_conn] = dace.pointer(array.dtype)
 
 
+class BlockVerticalLoopExpander(NaiveVerticalLoopExpander):
+
+    default_tile_sizes = (64, 8)
+
+    def get_tiled_subset_strs(self, nsdfg, extent):
```

Comment:
It's still quite difficult for me to parse this code without type hints. Is this going to be replaced with a new version that has those? If not, your IDE and the next reader through here would appreciate them!